NEWS.md

```diff
@@ -0,0 +1,40 @@
+
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+### Added
+
+### Changed
+- Refactored HistogramZoo functions to take in `Histogram` objects as opposed to numeric vectors
+- Estimation of summary statistics of histogram data (mean, standard deviation and skew)
+
+### Fixed
+
```

Comment:
Can you bump the version number to 1.3.0 and add these changes under the 1.3.0 section?
1.2.0 is merged into main but doesn't have a section since I merged before there was a NEWS file.